net/https-dns-proxy/files/https-dns-proxy.init

```diff
@@ -5,6 +5,8 @@
 export START=80
 export USE_PROCD=1
 
+export dnsmasqConfig
```

Comment:
Why do we need to export this?

Reply:
It was added to avoid Shellcheck SC2154 warning.